src/executor/attach_op_execs_pass.cc

```diff
@@ -23,42 +23,85 @@ const OperatorProperty* OpPropGetOpProperty(const NodeAttrs& attrs);
 
 namespace exec {
 
-// stateful compute executor
-class StatefulComputeExecutor : public OpExecutor {
+// abstract OpExecutor which provides storage fallback procedure on
```

Comment:
It didn't implement `Run()` which is pure virtual. I'll mention the two subclasses inherit from this class in comment.
https://github.com/eric-haibin-lin/mxnet/blob/470207345fdfb0917cefd53ce0d689dd3e639ea3/src/executor/exec_pass.h#L59